internal/cli/models.go

```diff
@@ -192,6 +192,26 @@ type (
 		ApprovalNotification *ApprovalNotification `json:"approvalNotification,omitempty"`
 	}
 
+	ActionExecutePermissions struct {
+		Users       []string                `json:"users"`
+		Roles       []string                `json:"roles"`
+		Teams       []string                `json:"teams"`
+		OwnedByTeam *bool                   `json:"ownedByTeam"`
+		Policy      *map[string]interface{} `json:"policy"`
+	}
+
+	ActionApprovePermissions struct {
+		Users  []string                `json:"users"`
+		Roles  []string                `json:"roles"`
+		Teams  []string                `json:"teams"`
+		Policy *map[string]interface{} `json:"policy"`
```

Comment:
```suggestion
		Policy *map[string]any `json:"policy"`
```
its an alias and makes more sense imo